localstack/services/transcribe/provider.py

```diff
@@ -136,7 +139,16 @@ def start_transcription_job(
         if not output_key:
             output_key = f"{job_name}.json"
 
-        transcript = Transcript(TranscriptFileUri=f"s3://{output_bucket}/{output_key}")
+        s3_client = aws_stack.connect_to_service("s3")
```

Comment:
Use the new client here

```suggestion
        s3_client = connect_to().s3
```